lib/code-block-writer.d.ts

```diff
@@ -1,210 +1,210 @@
-interface CodeBlockWriterOptions {
```

Comment:
I wonder why git said this file changed. I don't see any changes to line endings when I open it in a diff. Oh well, not a big deal.